package.json

```diff
@@ -46,17 +52,19 @@
     "file-saver": "^2.0.2",
     "husky": "^4.3.0",
     "jest": "^26.4.2",
+    "jest-transform-svelte": "^2.1.1",
     "license-checker": "^25.0.1",
     "mini-css-extract-plugin": "^0.11.2",
     "moment": "^2.29.0",
     "normalize.css": "^8.0.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.1.2",
     "prettier-plugin-svelte": "^1.4.0",
+    "react-is": "^16.13.1",
```

Comment:
For what is `react-is` needed? 🙂

Reply:
I think it got installed automatically with Storybook. It looks like a testing tool so if you don't plan on using jest it should be safely removed. I'll check whether it's needed at all.